Tools/Scripts/webkitpy/w3c/test_exporter_unittest.py

```diff
@@ -150,8 +152,10 @@ def test_export(self):
 
     def test_export_with_specific_branch(self):
         host = TestExporterTest.MyMockHost()
+        host.web.responses.append({'status_code': 200, 'data': {'login': 'USER'}})
         options = parse_args(['test_exporter.py', '-g', 'HEAD', '-b', '1234', '-c', '-n', 'USER', '-t', 'TOKEN', '-bn', 'wpt-export-branch'])
-        exporter = TestExporter(host, options, TestExporterTest.MockGit, TestExporterTest.MockBugzilla, MockWPTGitHub, TestExporterTest.MockWPTLinter)
+        w3c_changeset = W3cChangeset(host, options)
+        exporter = TestExporter(host, options, w3c_changeset, TestExporterTest.MockGit, TestExporterTest.MockBugzilla, MockWPTGitHub, TestExporterTest.MockWPTLinter)
```

Comment:
Creating the `W3cChangeset` then injecting it to `TestExporter` is a bit repetitive. @nomadtechie do you have any thought on if this should be cleaned up at this point?